src/dashboard/pages/index/index.py

```diff
@@ -1,11 +1,109 @@
 """Index page module."""
 import dash
 from dash import html
-from dash.dependencies import Component
+
+from dashboard.components.icon import icon
 
 dash.register_page(__name__, path="/", name="Home", order=0, nav_item=True, icon_name="home")
 
+default_style = "bg-[#d2d2d2] transition duration-500 ease-in-out"
+
+# This username will be replaced later
+username = "cooluser"
+LATEST_CONTAINER = "Latest opened dashboards"
+SHARED_CONTAINER = "Shared dashboards"
+
+
+def carousel_layout(container_title: str, id: str = None) -> html.Div:
```

Comment:
Mypy complains about default. `id` should have type `Optional[str]` or `str | None`.